data.sql

```diff
@@ -9,4 +9,30 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', '1998-10-13', 3, true, 17);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '2022-05-14', 4, true, 22);
+
+-- Insert data in owners table --
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', '34'),('Jennifer Orwell', '19'),
+	('Bob', '45'),
+	('Melody Pond', '77'),
+	('Dean Winchester', '14'),
+	('Jodie Whittaker', '38');
+
+-- Insert data into species table -- 
+INSERT INTO species(name)VALUES('Pokemon','Digimon');
+
```

Comment:
- [ ] The syntax you used here is incorrect and it causes an error when I try to insert values into the `species` table. This syntax looks like you want to give two values for the same entry and it would produce the error as seen below:

![image](https://user-images.githubusercontent.com/77106962/235735280-c30db355-33f5-4903-abd4-99a5bf35e6f0.png)

Kindly correct the syntax. 👍 😎

Reply:
Corrected the syntax as suggested above:
![Screenshot from 2023-05-03 09-32-52](https://user-images.githubusercontent.com/118063058/235861216-466ce09f-f83c-48a5-a90a-c5c2e01a0a61.png)

Reply:
Fixed.